### DISCUSSION TITLE
Statistics/Reports/Analytics

### DISCUSSION THREAD

**Discussion by missincrypto** [jagrosh/Vortex#123] [Questions]
Good Maen all. 

The discord server I cmod in uses Vortex heavily. Love it. 

Is there anyway to look at usage? As part of our weekly report, I'd love to be able to include how often Vortex was used, especially for the strike system. As you likely know, mod work happens in the shadows and I would love to have some numbers that show what's happening to the "outside" world. 

Thanks!

**Comment by MichailiK** [comment]
Vortex is designed to store as little data as (reasonably) possible. In this case, Vortex only uses a simple counter for how many strikes someone has, nothing else. Vortex does not store usages or any other statistics/metadata.

You could try using the search feature of Discord instead.
